TextFormat.md

```diff
@@ -34,7 +34,35 @@ represented as hexadecimal floating-point as specified by the C99 standard, whic
 IEEE-754-2008 section 5.12.3 also specifies. The textual format may be improved to also
 support more human-readable representations, but never at the cost of accurate representation.
 
-## Debug symbol integration
+# Official Text Format
+
+WebAssembly currently doesn't have a final, official, text format. As detailed above the
+main purpose of the text format will be for human consumption, humans feedback on readability
```

Comment:
either `human` or `humans'`, I think. but `feedback from humans` might be clearer